### PR TITLE
0137 math doc

### LINKED ISSUE #137: Document maths of covid predictions

### PR DESCRIPTION
**Pull request by jaryaman**
Details of the mathematical model used in `covidvu.predict`

@kinabalu you can view the file if you paste the URL into here: https://nbviewer.jupyter.org/ It doesn't render properly in github (though nbviewer seems to be stuck on an older version).

### CODE REVIEW

**Comment by pr3d4t0r** [comment]
@jaryaman - This looks good, thanks.  I saw Andrew and I are both marked as reviewers, but you addressed him directly in the pull request comment; I'll wait for him to review/approve/merge.

I opened the GitHub's wiki page, made the landing page an empty FAQ...

File: work/prediction-notes.ipynb [hunk 1]
```diff
@@ -0,0 +1,134 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# COVIDvu <img src='resources/UN-flag.png' align='right'>\n",
+    "\n",
+    "COVID-19 model for predicting future outcomes.\n",
```
Comment (pr3d4t0r) [inline work/prediction-notes.ipynb]: Would this apply here?

_COVID-19 model for predicting the rate at which the virus burn through the population._

Or something like that?

Comment (pr3d4t0r) [reply work/prediction-notes.ipynb]: "Burn" as in epidemiology jargon for a virus epidemic infecting all suceptinble subjects until it runs out.

**Comment by jaryaman** [comment]
@pr3d4t0r happy for you to merge -- just thought I'd try and give Andrew a convenient way of viewing the notebook. Though nbviewer seems to have its own bugs too.

File: work/prediction-notes.ipynb (hunk 1 above)
Comment (jaryaman) [reply work/prediction-notes.ipynb]: How about

_COVID-19 model for predicting the spread of coronavirus through populations._?

Struggled to make a title with the word "burn" that wasn't too verbose.

**Pull request merged**